__tests__/app.test.js

```diff
@@ -416,3 +418,93 @@ describe("GET /api/users", () => {
     })
 })
 
+describe("GET /api/articles (topic query)", () => {
+    test("responds with articles filtered by given topic query", () => {
+        return request(app).get('/api/articles/?topic=cats')
+        .expect(200)
+        .then(({ body }) => {
+            expect(body.articles.length).toBe(1)
+            body.articles.forEach((article) => {
+                expect(article).toMatchObject({
+                    author: expect.any(String),
+                    title: expect.any(String),
+                    article_id: expect.any(Number),
+                    topic: 'cats',
+                    created_at: expect.any(String),
+                    votes: expect.any(Number),
+                    article_img_url: expect.any(String),
+                    comment_count: expect.any(Number)
+                })
+            })
+            expect(body.articles).toBeSortedBy('created_at', {descending: true})
```

Comment:
We've already asserted in a previous test how the results are sorted so no need to duplicate it here, plus in this case we aren't gaining anything as there's only one response.